Gemfile

```diff
@@ -1,5 +1,8 @@
 source 'https://rubygems.org'
 
+# rails-assets requires >= 1.8.4
+gem 'bundler', '>= 1.8.4'
```

Comment:
Aah true..
But they have explicitly mentioned to include:
`gem 'bundler', '>= 1.8.4'`
in the Gemfile

![screenshot from 2016-03-08 20 32 09](https://cloud.githubusercontent.com/assets/7537349/13605341/2193ad9a-e56d-11e5-9fd3-32423de5ef2e.png)